### DISCUSSION TITLE
vcpkg use only 1 core...

### DISCUSSION THREAD

**Discussion by Dariusz1989** [microsoft/vcpkg#26174] [Q&A]
Hey

How can I tell vcpkg to pass multi thread flag to nunja/etc whatever is using? Its building everything using 1 core, 1x cl.exe :- ((

Regards
Dariusz

**Comment by JackBoosY** [comment]
It depends on `VCPKG_CONCURRENCY`.
You can manually set it as the number you need in the triplet file.

**Comment by dg0yt** [comment]
Vcpkg normally uses all available cores, and *you don't have to do anything* unless concurrency leads to problems.
However, some ports may be unable to make good use of it.
